real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -65,6 +65,8 @@
 #define BRAKE_INDICATOR_PORT PORTE
 #define BRAKE_INDICATOR_PINN PE3 // arduino 5
 
+#define STATE_THRESHOLD 120 //This is a temporary change to eliminate magic number
```

Comment:
They are the same. It is a threshold used to make a pwm (servo) input binary. Can we name it PWM_STATE_THRESHOLD so it is grouped with the other PWM defines above?